hello-world/pom.xml

```diff
@@ -1,137 +1,181 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
+    <modelVersion>4.0.0</modelVersion>
 
-  <groupId>org.acme</groupId>
-  <artifactId>hello-world-school-timetabling</artifactId>
-  <version>1.0-SNAPSHOT</version>
+    <groupId>org.acme</groupId>
+    <artifactId>hello-world-school-timetabling</artifactId>
+    <version>1.0-SNAPSHOT</version>
```

Comment:
These changes are unrelated and unnecessary. Probably your IDE messed up?
Please remove from this file and others.